language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -549,6 +555,11 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
         tyvec: &[mty::Type],
         linkage: llvm::LLVMLinkage,
     ) {
+        debug!(target: "sbc", "Move bytecode for function {:?}", &fn_env.get_name_str());
+        for bc in fn_env.get_bytecode() {
+            debug!(target: "sbc", "bytecode {:?}", bc);
+        }
+
```

Comment:
The above prints an entire line `2023-06-13 02:04:08.128673508] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:558 [DEBUG]` for _every_ single bytecode instruction. This is too much noise-to-signal. Also, you are printing Move bytecode, not stackless byte code. The "sbc" in target is when the developer wants to dump information related to stackless bytecode. Perhaps use "mbc" as target. Overall, something like this:
`debug!(target: "mbc", "Move stack bytecode for function {:?}\n{:#?}", &fn_env.get_name_str(),fn_env.get_bytecode());`

This way, you get only one information line for the whole list of Move bytecode. Note that we already pretty print the actual stackless bytecode for each function, which is the form we are translating from. This includes the stackless bytecode instructions as well as the types of all the locals.
```
move-mv-llvm-compiler/src/stackless/translate.rs:867 [DEBUG] 
public fun <SELF>::<SELF>() {
     var $t0|$t0: u8
     var $t1|$t1: u128
     var $t2|$t2: ST::A1
     var $t3|$t3: ST::A1
     var $t4|$t4: vector<ST::A1>
     var $t5: u8
     var $t6: u128
     var $t7: ST::A1
     var $t8: u8
     var $t9: u128
     var $t10: ST::A1
     var $t11: &ST::A1
     var $t12: vector<ST::A1>

```

Reply:
I'll change it to one `debug!` printing, it will produce something like:
```
[2023-06-13 03:38:18.545980542] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:562 [DEBUG] Move stack bytecode for function "moo"
[
    LdU32(7),
    Pack(0),
    Pack(1),
    StLoc(0),
    ImmBorrowLoc(0),
    ImmBorrowField(FieldHandleIndex(0)),
    ImmBorrowField(FieldHandleIndex(1)),
    ReadRef,
    Ret,
]
```